server/devdocs/selectors/index.js

```diff
@@ -23,47 +29,105 @@ const SELECTORS_DIR = path.resolve( __dirname, '../../../client/state/selectors'
 const router = express.Router();
 let prepareFuse;
 
-function parseSelectorFile( file ) {
-	return new Promise( ( resolve, reject ) => {
-		fs.readFile( SELECTORS_DIR + '/' + file, 'utf8', ( error, contents ) => {
-			if ( error ) {
-				return reject( error );
+const jsParse = contents => {
+	try {
+		return espree.parse( contents, {
+			attachComment: true,
+			ecmaVersion: 6,
```

Comment:
We might want to bump this up since we support a few features that aren't available in ES2015. To a more general point, we should consider using a parser that's consistent with our build ([Babylon](https://github.com/babel/babylon)).